app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controllers");
+const app = express();
+
```

Comment:
The line of code below isn't needed until we test a post request. Please be careful to only include lines of code which are necessary to complete the ticket in hand